NetherLoadTest/NetherLoadTest/NetherUnitTest.cs

```diff
@@ -122,6 +122,25 @@ public void Init()
             TestContext.EndTimer("GetScore");
         }
 
+        [TestMethod]
+        public async Task playGame()
+        {
+            // simuate game - users logs in, get the scores from the leaderbaord 
+            // after a random wait time the user will post the new score and get the leaderboard again. 
+            await EnsureLoggedInAsync();
+
+            TestContext.BeginTimer("GetScore");
+            await _client.GetScoresAsync();
+            TestContext.EndTimer("GetScore");
+            // sleep between 30 seconds to 5 minutes
+            Thread.Sleep(s_random.Next(30, 300) * 1000);
+
+            TestContext.BeginTimer("PlayLevel");
```

Comment:
I think it's probably better to put separate times around the PostScore and GetScores here - will probably make it easier to see what it happening